data/data/aws/master/main.tf

```diff
@@ -99,10 +99,10 @@ resource "aws_instance" "master" {
     ignore_changes = ["ami"]
   }
 
-  # `clusterid` is required for machine-api
+  # `kubernetes.io/cluster/*` is required for machine-api
   tags = "${merge(map(
     "Name", "${var.cluster_id}-master-${count.index}",
-    "clusterid", "${var.cluster_id}",
+    "kubernetes.io/cluster/${var.cluster_id}", "owned",
```

Comment:
this is not required it is already provided through the root/main.tf

Reply:
Good point thanks.